README.md

```diff
@@ -22,6 +22,16 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 * Edge (20+)
 * Safari (7+)
 
+## Dependencies
+* [Angular 2](https://angular.io) (tested with 2.0-rc.4)
+* [Bootstrap 4](https://v4-alpha.getbootstrap.com) (test with 4.0 alpha V2)
+
+## Installation
+As soon as you've got the above dependencies installed, install `ng-bootstrap` via:
+```
+npm install --save @ng-bootstrap/ng-bootstrap
+```
+
```

Comment:
We should also describe how to:
* import directives
* add directives that need to be pre-comopiled